docs/book/src/reference/versions.md

```diff
@@ -95,6 +95,16 @@ get all the fixes/latest changes.
 
 In case of upgrades, clusterctl should be upgraded first and then used to upgrade all the other components.
 
+#### coreDNS upgrades
+Newer versions of CoreDNS may not be compatible as an upgrade target for clusters managed with older versions of Cluster API. 
+Below is a table showing which CoreDNS versions associated with that release of Kubernetes are supported in Cluster API upgrades.
+
```

Comment:
I think the CoreDNS versions we can upgrade to are independent of the Kubernetes version. It only depends on the version of the CoreDNS lib in the respective CAPI version.

Reply:
I'm trying to get across the Kubeadm/CoreDNS association here as a guide to compatibility (not sure what the K8s/CoreDNS matrix is like). Should I remove the Kubernetes version altogether?

Reply:
I'm not sure, let's discuss this tomorrow with Fabrizio. Just to avoid back and forth :)

Reply:
Let's talk about this next week to get a quick consensus